Monika After Story/game/sprite-chart.rpy

```diff
@@ -3790,6 +3790,7 @@ init -3 python:
                     acs_layer
                 )
 
+        @store.mas_utils.deprecated()
```

Comment:
the `wear_acs_x` replacement is `wear_acs_in` with the appropriate ACS constant.